x-pack/plugins/transform/public/app/hooks/use_index_data.ts

```diff
@@ -84,21 +84,21 @@ export const useIndexData = (
       return;
     }
 
-    const isCrossClusterSearch = indexPattern.title.includes(':');
+    const isCrossClusterSearch = dataView.title.includes(':');
     const isMissingFields = resp.hits.hits.every((d) => typeof d.fields === 'undefined');
 
     const docs = resp.hits.hits.map((d) => getProcessedFields(d.fields ?? {}));
 
     // Get all field names for each returned doc and flatten it
     // to a list of unique field names used across all docs.
-    const allKibanaIndexPatternFields = getFieldsFromKibanaIndexPattern(indexPattern);
+    const allKibanaDataViewFields = getFieldsFromKibanaIndexPattern(dataView);
```

Comment:
`KibanaIndexPattern` was used to distinguish between raw string based index patterns and a Kibana maintained index pattern object. So suggest to now name this just `allDataViewFields` since `DataView` is a Kibana-only thing anyway. I see `getFieldsFromKibanaIndexPattern` is brought over from the `ml` plugin so this will be done in another PR?

Reply:
Renamed to `allDataViewFields` in https://github.com/elastic/kibana/pull/128247/commits/b846d838066b7befa01f487a1d1caa88bcfeffbd. Yes I have left the remaining `IndexPattern` to `DataView` function / variable renaming in the `ml` plugin for a follow-up :)